src/Components/Profile/Profile.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import logo from "./2222.jpg";
```

Comment:
Consider following the import comments above.

Reply:
i've added new line between imports
![image](https://user-images.githubusercontent.com/46630057/74604813-7e449700-50d2-11ea-92f5-a45aacd774bc.png)